### PR TITLE
UI/Layout fixes

### PR DESCRIPTION
**Pull request by SebaUbuntu**

### CODE REVIEW

**Comment by SebaUbuntu** [comment]
Everything seems to work with these changes, merge?

**Comment by luk1337** [comment]
`recording = ` is delayed by 500ms when shutter sounds are active, within that time primary/secondary buttons are still enabled. Perhaps this should be addressed.

File: app/src/main/java/org/lineageos/selfie/MainActivity.kt [hunk 1]
```diff
@@ -537,74 +525,78 @@ class MainActivity : AppCompatActivity() {
...
 
     /**
      * Check if we can reinitialize the camera use cases
      */
     private fun canRestartCamera() = when (cameraMode) {
         // Disallow camera restart if we're taking a photo or if timer is running
-        CameraMode.PHOTO -> !isTakingPhoto && !countDownView.isVisible
+        CameraMode.PHOTO -> !takingSomething && !countDownView.isVisible
         // Disallow camera restart if a recording in progress or if timer is running
         CameraMode.VIDEO -> !cameraController.isRecording && !countDownView.isVisible
```
Comment (SebaUbuntu) [inline app/src/main/java/org/lineageos/selfie/MainActivity.kt]: This can now match photo mode conditions (`!takingSomething && !countDownView.isVisible`)

**Comment by SebaUbuntu** [comment]
> `recording = ` is delayed by 500ms when shutter sounds are active, within that time primary/secondary buttons are still enabled. Perhaps this should be addressed.

Fixed

**Comment by luk1337** [comment]
I think it might be a good idea to have one common variable that stores current state:
- idle
- taking photo
- recording
- recording paused

**Comment by luk1337** [comment]
@SebaUbuntu pushed some of your changes to main branch + rebased everything else.

File: app/src/main/java/org/lineageos/selfie/MainActivity.kt [hunk 2]
```diff
@@ -587,67 +586,65 @@ class MainActivity : AppCompatActivity() {
...
-                videoDuration.text = TimeUtils.convertNanosToString(duration)
-                videoDuration.isVisible = enabled
-
-                // Update video recording pause/resume button visibility state
-                if (duration == 0L) {
-                    flipCameraButton.isInvisible = enabled
-                    videoRecordingStateButton.isVisible = enabled
+        val delayTime = if (cameraSoundsUtils.playStartVideoRecording()) 500L else 0L
+
+        shutterButton.postDelayed({
```
Comment (luk1337) [inline app/src/main/java/org/lineageos/selfie/MainActivity.kt]: wouldn't we want handler.postDelayed() here?

Comment (SebaUbuntu) [reply app/src/main/java/org/lineageos/selfie/MainActivity.kt]: Done

**Pull request merged**